### PR TITLE
refactor(build): Move top level provider config to com.spinnaker.config

### PR DESCRIPTION
**Pull request by dibyom**
Any Spring Configuration under com.spinnaker.config will be scanned by
`clouddriver-web` removing the necessity to specify them directly using
an `Import`.

### CODE REVIEW

File: clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/config/AppengineConfiguration.groovy [hunk 1]
```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2016 Google, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/config/AppengineConfiguration.groovy]: I would leave this date the same

File: clouddriver-appengine/src/main/java/com/netflix/spinnaker/config/AppengineStorageConfiguration.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Google, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-appengine/src/main/java/com/netflix/spinnaker/config/AppengineStorageConfiguration.java]: Silly intellij

File: clouddriver-artifacts/src/main/java/com/netflix/spinnaker/config/ArtifactConfiguration.java [hunk 3]
```diff
@@ -1,22 +1,22 @@
 /*
- * Copyright 2017 Google, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-artifacts/src/main/java/com/netflix/spinnaker/config/ArtifactConfiguration.java]: Ping

File: clouddriver-aws/src/main/groovy/com/netflix/spinnaker/config/AwsConfiguration.groovy [hunk 4]
```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2015 Netflix, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-aws/src/main/groovy/com/netflix/spinnaker/config/AwsConfiguration.groovy]: :arrow_up:

File: clouddriver-azure/src/main/groovy/com/netflix/spinnaker/config/AzureConfiguration.groovy [hunk 5]
```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2015 The original authors.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-azure/src/main/groovy/com/netflix/spinnaker/config/AzureConfiguration.groovy]: :thinking:

File: clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/config/CloudFoundryConfiguration.java [hunk 6]
```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2018 Pivotal, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/config/CloudFoundryConfiguration.java]: :thinking:

File: clouddriver-core/src/main/groovy/com/netflix/spinnaker/config/CloudDriverConfig.groovy [hunk 7]
```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2015 Netflix, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-core/src/main/groovy/com/netflix/spinnaker/config/CloudDriverConfig.groovy]: :thinking:

File: clouddriver-core/src/main/groovy/com/netflix/spinnaker/config/DeployConfiguration.groovy [hunk 8]
```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2014 Netflix, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-core/src/main/groovy/com/netflix/spinnaker/config/DeployConfiguration.groovy]: :thinking:

File: clouddriver-core/src/main/groovy/com/netflix/spinnaker/config/LocalJobConfig.groovy [hunk 9]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Google, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-core/src/main/groovy/com/netflix/spinnaker/config/LocalJobConfig.groovy]: :thinking:

File: clouddriver-core/src/main/groovy/com/netflix/spinnaker/config/RetrofitConfig.groovy [hunk 10]
```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2015 Netflix, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-core/src/main/groovy/com/netflix/spinnaker/config/RetrofitConfig.groovy]: :thinking:

File: clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/config/DcosConfiguration.groovy [hunk 11]
```diff
@@ -1,7 +1,7 @@
 /*
- * Copyright 2017 Cerner Corporation
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/config/DcosConfiguration.groovy]: :thinking:

File: clouddriver-docker/src/main/groovy/com/netflix/spinnaker/config/DockerRegistryConfiguration.groovy [hunk 12]
```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2016 Google, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-docker/src/main/groovy/com/netflix/spinnaker/config/DockerRegistryConfiguration.groovy]: :thinking:

File: clouddriver-ecs/src/main/java/com/netflix/spinnaker/config/EcsConfiguration.java [hunk 13]
```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2018 Lookout, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-ecs/src/main/java/com/netflix/spinnaker/config/EcsConfiguration.java]: :thinking:

File: clouddriver-eureka/src/main/groovy/com/netflix/spinnaker/config/EurekaProviderConfiguration.groovy [hunk 14]
```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2016 Google, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-eureka/src/main/groovy/com/netflix/spinnaker/config/EurekaProviderConfiguration.groovy]: :thinking:

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/config/GoogleConfiguration.groovy [hunk 15]
```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2015 Google, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/config/GoogleConfiguration.groovy]: :thinking:

File: clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/config/KubernetesConfiguration.groovy [hunk 16]
```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2015 Google, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/config/KubernetesConfiguration.groovy]: :thinking:

File: clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/config/OpenstackConfiguration.groovy [hunk 17]
```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2016 Veritas Technologies LLC.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/config/OpenstackConfiguration.groovy]: :thinking:

File: clouddriver-oracle/src/main/groovy/com/netflix/spinnaker/config/OracleConfiguration.groovy [hunk 18]
```diff
@@ -1,12 +1,19 @@
 /*
- * Copyright (c) 2017 Oracle America, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-oracle/src/main/groovy/com/netflix/spinnaker/config/OracleConfiguration.groovy]: :thinking:

File: clouddriver-titus/src/main/groovy/com/netflix/spinnaker/config/TitusConfiguration.groovy [hunk 19]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Netflix, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (lwander) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/config/TitusConfiguration.groovy]: :thinking:

**Comment by dibyom** [comment]
I think I have that set. Looks like the license got overwritten only on the files that got moved to a new package.

**Pull request merged**